src/user/importexport-app.md

```diff
@@ -133,49 +132,104 @@ Import.
 > Metadata, and to check for problems with out-of-sync data elements or
 > organisation unit names
 
-#### PDF Data { #importPDFdata } 
+#### PDF Data { #importPDFdata }
 
 DHIS2 supports import of data in the PDF format. This can be used to
 import data produced by off-line PDF data entry forms. Please refer to
 the section **Data set management** for details on how to produce a PDF
 form which can be used for off-line data entry.
 
-To import a PDF data file, navigate to the *PDF Data Import* item in the
-side menu. Upload the completed PDF file and click *Import*.
+To import a PDF data file, navigate to the _PDF Data Import_ item in the
+side menu. Upload the completed PDF file and click _Import_.
 
-### Event Import { #event_import } 
+### Event Import { #event_import }
 
-Event can be access from the sidebar by clicking on Event
-import.
+Event Import can be accessed from the sidebar by clicking on Event
+Import.
 
 ![](resources/images/import_export/event_import.png)
 
-1.  Select a format: *JSON* , *CSV*, or *XML*
+1.  Select a format: _JSON_ , _CSV_, or _XML_
 
 2.  Click **Advanced options** if you want to adjust one or more of
     the following settings before importing:
 
-    * Event ID scheme
-    * Data element ID scheme
-    * Org unit ID scheme
-    * ID scheme
+    - Event ID scheme
+    - Data element ID scheme
+    - Org unit ID scheme
+    - ID scheme
 
 3.  Click on the **Import** button which will upload the file and start the
     importing process.
 
-### GML Import { #gml_import } 
+### Earth Engine Import { #ee_import }
 
-GML Import can be accessed from the sidebar by clicking on GML
+Earth Engine Import can be access from the sidebar by clicking on Earth Engine
 Import.
 
-![](resources/images/import_export/gml_import.png)
+Google Earth Engine (WorldPop) data is fetched using the Google Earth Engine API. A Google
+EarthEngine account is required to use this importer.
```

Comment:
Add space between "Earth Engine" + add a link to https://docs.dhis2.org/en/topics/tutorials/google-earth-engine-sign-up.html